Dockerfile.ci

```diff
@@ -1,20 +1,30 @@
-ARG commit=develop
+ARG image_tag=develop
 FROM elifesciences/proofreader-php:0.2 AS proofreader
-FROM elifesciences/annotations_cli:${commit}
-ENV DEBUG=1
+FROM elifesciences/annotations_composer_dev:${image_tag} AS build
+FROM elifesciences/php_cli:d83fc4714914898b0842199578e1cc88d9feab2a
+
+ENV PROJECT_FOLDER=/srv/annotations
 
 USER root
-RUN mkdir -p build && chown www-data:www-data build && \
-    touch .php_cs.cache && chown www-data:www-data .php_cs.cache
+RUN docker-php-ext-install pcntl
+
+WORKDIR ${PROJECT_FOLDER}
+RUN mkdir -p build var/logs var/cache/html_purifier && \
+    touch .php_cs.cache && \
+    chown --recursive elife:elife . && \
```

Comment:
(`--recursive`) shouldn't be needed? Where files are created that are not owned by `elife`?